### PR TITLE
sail: Bump version to 0.9.0-rc2

### PR DESCRIPTION
**Pull request by HappySeaFox**
Specify library name and version:  **sail/0.9.0-rc1**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated. NO because of https://github.com/conan-io/hooks/issues/157

### CODE REVIEW

File: recipes/sail/all/conanfile.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
+# pylint: skip-file
```
Comment (uilianries) [inline recipes/sail/all/conanfile.py:1]:
```suggestion
```
The linter is mandatory on the Conan recipe.

File: recipes/sail/all/conanfile.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
 from conan.tools.files import rename
 from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
```
Comment (uilianries) [inline recipes/sail/all/conanfile.py:4]:
```suggestion
from conan.errors import ConanInvalidConfiguration
```
Use the new namespace: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#python-import-statements

File: recipes/sail/all/conanfile.py [hunk 3]
```diff
@@ -56,6 +58,11 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
+    # TODO Probably will not be needed in Conan 2.x https://github.com/conan-io/conan-center-index/issues/13001
+    #def validate(self):
+    #    if self.options.with_avif and tools.cross_building(self):
+    #        raise ConanInvalidConfiguration("Cross-building is not implemented due to dependencies (dav1d)")
+
```
Comment (uilianries) [inline recipes/sail/all/conanfile.py:65]:
```suggestion
```
Please, remove commented code. Git log can show the past

**Review by uilianries — COMMENTED** [review]
Thank you for your contribution. See that we are updating Conan recipes to Conan 2.0. We have a migration guide documented here: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#migrating-the-recipes

- Please, update required_conan_version = ">=1.43.0" to 1.51.3

**Comment by HappySeaFox** [comment]
Waiting for https://github.com/conan-io/conan-center-index/pull/13002

**Comment by HappySeaFox** [comment]
This package is not ready yet for Conan V2. I tried to migrate, but there were linking errors. I have no time to investigate more, so the package is v1 still.

**Comment by uilianries** [comment]
@HappySeaFox Please, update libtiff from conanfile.py to the version `4.4.0`

**Comment by HappySeaFox** [comment]
> @HappySeaFox Please, update libtiff from conanfile.py to the version `4.4.0`

Unfortunately, there are new errors:

```
ERROR: Missing binary: jasper/2.0.33:4bce3da051cd18778517e04ea89cab39d6ca88fc
ERROR: Missing binary: libyuv/1841:5933d3a07b59c9a66174e954ca8dc92324261d6b
```

**Comment by HappySeaFox** [comment]
@uilianries @SpaceIm 

Could you please help understanding why the CMake project cannot find libavif (using `find_package(libavif)`): https://c3i.jfrog.io/c3i/misc/logs/pr/13000/38-linux-gcc/sail/0.9.0-rc1//b2db50554ae269b750144626b21faed36d26d12d-build.txt

Previously it worked in Conan V1.

**Comment by SpaceIm** [comment]
> @uilianries @SpaceIm
> 
> Could you please help understanding why the CMake project cannot find libavif (using `find_package(libavif)`): https://c3i.jfrog.io/c3i/misc/logs/pr/13000/38-linux-gcc/sail/0.9.0-rc1//b2db50554ae269b750144626b21faed36d26d12d-build.txt
> 
> Previously it worked in Conan V1.

You have forgotten CMakeDeps generator.

**Comment by HappySeaFox** [comment]
@uilianries 

Jasper is currently blocked by https://github.com/conan-io/conan-center-index/issues/13159

**Comment by uilianries** [comment]
@HappySeaFox Please, consider the pull request https://github.com/HappySeaFox/conan-center-index/pull/1

**Comment by HappySeaFox** [comment]
Yesterday I opened another PR with the SDK workaround and rc2: https://github.com/conan-io/conan-center-index/pull/14871 Let's continue there.

**Pull request closed without merging**